docs/usage.md

```diff
@@ -6,28 +6,202 @@
 
 ## Introduction
 
-The TreeVal pipeline has three requirements before being able to run
+The TreeVal pipeline has a few requirements before being able to run:
 
-Firstly, it requires a .yaml file (described below in Full Samplesheet). This will contain all of the information required for running the TreeVal pipeline.
+- The gene_alignment_data and synteny data much follow a particular directory structure
 
-The cram files must each be accompanied by an index file (.crai) generated by samtools index.
+- HiC CRAM files much already be pre-indexed in the same location as the CRAM file, e.g., `samtools index {cram file}`. If this would be more helpful to the community as an automated process then please submit an issue.
 
-The gene alignment data is also expected to follow a particular folder structure, such as (using data from the yaml below):
+- Finally, the yaml file which is described below in Full Samplesheet. This needs to contain all of the information related to the assembly for the pipeline to run.
+
+### Prior to running TreeVal
+
+:warning: Please ensure you read the following sections on Directory Strucutre (gene_alignment_data, synteny, scripts), HiC data prep and Pacbio data prep. Without these you may not be able to successfully run the TreeVal pipeline. If nothing is clear then leave an issue report.
+
+#### Directory Structure
+
+Working example found here, [EXAMPLE](genealignmentsynteny.md), this will cover setting up synteny and gene_alignment_data directories as well as downloading some example data.
```

Comment:
```suggestion
Working example found [here](genealignmentsynteny.md), this will cover setting up synteny and gene_alignment_data directories as well as downloading some example data.
```